Detectors/ITSMFT/common/reconstruction/src/Cluster.cxx

```diff
@@ -81,7 +81,7 @@ void Cluster::setPatternColSpan(UShort_t nc, Bool_t truncated)
 }
 
 #endif
-
+/*
 //______________________________________________________________________________
```

Comment:
@iouribelikov : seems to be due to the conflict with my previous PR.

Reply:
@shahor02 I am looking into it...